hw03_wange2.py

```diff
@@ -0,0 +1,73 @@
+# Name: Eun Sung Wang
```

Comment:
Use our template header block for formatting the header.

Reply:
I have added them.